apps/server/src/utils/parserFunctions.ts

```diff
@@ -216,20 +221,67 @@ export const parseOsc = (data: { osc?: Partial<OSCSettings> }): OSCSettings => {
   }
 };
 
+/**
+ * Parses and validates HTTP subscription cycle options
+ * @param data
+ */
+export const validateHttpSubscriptionCycle = (data: HttpSubscriptionOptions[]): boolean => {
+  for (const subscriptionOption of data) {
+    const isHttp =
+      subscriptionOption.message?.startsWith('http://') || subscriptionOption.message?.startsWith('https://');
```

Comment:
would we not need a certificate for https?

have you added code to generate this? maybe we can leave https for next time

Reply:
we can 
but I don't think a client needs a certificate
https://security.stackexchange.com/questions/110621/ssl-newbie-does-https-client-also-need-a-certificate
https://stackoverflow.com/questions/19383538/what-is-the-difference-between-requiring-an-ssl-cert-and-accepting-an-ssl-cert

in my tests it also works fine.